src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -53,17 +53,14 @@ public void ValidPathExists_ReturnsTrue()
             });
         }
 
-        [Fact]
-        public void PathWithInvalidCharactersAsPath_ReturnsFalse()
+        [Theory MemberData("PathsWithInvalidCharacters")]
```

Comment:
Thanks for the heads up. Out of curiosity, why is it important to make commas mandatory?